lib/postprocessing/image.js

```diff
@@ -459,6 +473,29 @@ async function imagePostProcess(intermediateRendition, rendition, directories) {
     await write(img, rendition.path);
 }
 
+function convertSvg(img, instructions) {
+
+    // Note RSVG is the converter here currently 2020/10/22 causing differences between normal Imagemagic use
+    // Future - allow an instruction to define what color should be converted to transparent
+
+    // ImageMagick automatically keeps aspect ratio
+    // Only using width because ImageMagick will use the smallest value whether it be width or height
+    const width = instructions.width || 400;
+    const fmt = instructions.fmt;
+
+    if (NON_ALPHA_FMT.includes(fmt)) {
+        img.background("rgb(255,255,255)"); // Change transparency to white
+        img.flatten(); // forces the image on its background
+    } else {
+        img.background("rgba(0, 0, 0, 0.0)"); // Keep transparency
+    }
+
+    // some svgs have no size (only percentage width/height), so we scale them to our target size
+    img.in("-size", `${width}`)
+    img.colorspace('RGB');
```

Comment:
I'm curious - why do we need colorspace here?

Reply:
I think it was from a previous Alex comment ... but now I can't seem to find it
I'll take it out